modcc/printer/cprinter.cpp

```diff
@@ -806,6 +828,32 @@ void emit_simd_state_update(std::ostream& out,
                                data, index, weight, scaled);
         }
     }
+    else if (write_voltage) {
+        if (d.index_var_kind == index_kind::node) {
+            if (constraint == simd_expr_constraint::contiguous) {
+                out << fmt::format("indirect({} + {}, simd_width_) = {};\n",
+                                   data, node, name);
+            }
+            else {
+                // We need this instead of simple assignment!
+                // SAFETY: only one V-PROCESS per CV allowed
```

Comment:
It's used: If multiple V-PROCESSes were allowed on one compartment, eg
```
A: V = 42
B: V = 23
```
the outcome is undefined and/or racy (CPU/GPU). So, if only one is there, it's
ok to use this pattern to set V instead of building monoidal updates, like for 
currents, where
```
A: iX = 42
B: iX = 23
```
aggregate to `iX = 65`.

The guarantee is given by `fvm_layout`.

Reply:
ok, but is this comment only valid for the  `else` branch?

Reply:
Ah, that's just misplaced!

Reply:
And moved.